hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java

```diff
@@ -25,6 +27,12 @@
     @NotNull
     private ContentType contentType;
 
+    @Min(1024)
+    @Max(2 * 1024 * 1024)
```

Comment:
I can't see how this is configured in Configs. There is an option for `KAFKA_CONSUMER_MAX_PARTITION_FETCH_BYTES_CONFIG` but these values do not map 1-1 to each other for all cases.